tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true
```

Comment:
hvis dette er for å unngå switch-case uten break kan vi kanskje heller bare legge til det hvor det mangler (hvis det er det som er best-practice) https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-8.html#case-clause-fall-throughs

Reply:
ja, kan se litt mer på denne!

Reply:
Får denne feilen hvis jeg ikke har med den linja der: 

<img width="908" alt="Skjermbilde 2020-10-28 kl  12 49 26" src="https://user-images.githubusercontent.com/6595982/97432305-0c3c7700-191c-11eb-8b95-19617cf8043a.png">

Reply:
bug på det her: https://github.com/facebook/create-react-app/issues/9429